### ISSUE TITLE
Modifier Keys are not working in Safari (OSX), Firefox (OSX), and IE 11 (Win10)

### ISSUE THREAD

**Issue by JonathanBristow** [null-null-null/ember-keyboard#32]
I cannot get any keyboard shortcuts which contain a ctrl, alt or shift to work in the browsers mentioned. I am testing using a Mac (and Parallels for Windows 10/IE11).

The demo linked to in the read me doesn't work fully in Firefox, you can press the arrow keys but when you use the modifier key it doesn't respond.

Is this something you are aware of and is there a fix/workaround?

Thanks!

**Comment by null-null-null** [comment]
Thanks for the report, @JonathanBristow. I think this is related to https://github.com/null-null-null/ember-keyboard/issues/30. I'm gonna get my hands on a Mac soon, which hopefully will give me some insights into what's going on. Unfortunately, I can't replicate any of these bugs on Ubuntu, my own OS.

If you're interested in helping, what I really need is to look at the key event. You could grab it by setting a breakpoint [here](https://github.com/null-null-null/ember-keyboard/blob/master/addon/services/keyboard.js#L57). In particular, is `ctrlKey` true? And what about the other modifier keys? Also, is it a keydown, keyup, or keypress event?

**Comment by oliverwilkie** [comment]
I've noticed in my experience that

`shift+r` doesnt work in my Chrome but `shift+R` does

**Comment by oliverwilkie** [comment]
I think I know how to reproduce it

Imagine you have a listener for shift+r 

When you press shift+r together the gather keys method returns `shift` and `R`.
We would expect it to return `shift` and `r`

I think it can be fixed by editing `ember-keyboard/addon/utils/get-key.js` from
```
  function getKey(event) {
    return event.key || _emberKeyboardFixturesKeyMap['default'][event.keyCode];
  }
```
to
```
  function getKey(event) {
    return_emberKeyboardFixturesKeyMap['default'][event.keyCode] || event.key;
  }
```

**Comment by oliverwilkie** [comment]
Hastily prepared PR: https://github.com/null-null-null/ember-keyboard/pull/33

**Comment by JonathanBristow** [comment]
Hi Oliver 

I'll be picking this up again in about a week so I'll test it out and see what I can work out then.

Jon

**Comment by null-null-null** [comment]
I believe this issue has been resolved as of `v2.0.0`. Let me know if we should reopen this!
